features/step_definitions/user_steps.rb

```diff
@@ -42,39 +42,44 @@
 Given(/^I am logged into the site$/) do
   visit new_user_session_path
   FactoryGirl.create(:user)
-  fill_in 'Email', with: 'newuser@sb.com' 
-  fill_in 'Password', with: 'password' 
+  fill_in 'Email', with: 'newuser@sb.com'
+  fill_in 'Password', with: 'password'
   click_on 'Log in'
 end
 
 When(/^I visit my own profile page$/) do
-  pending # express the regexp above with the code you wish you had
+ visit root_path
 end
 
 Then(/^I see my username$/) do
-  pending # express the regexp above with the code you wish you had
+  expect(page).to have_content("NewUser")
 end
 
 Then(/^I can see my avatar$/) do
-  pending # express the regexp above with the code you wish you had
+  visit page.find('#avatar')[:src]
+  page.status_code.should be 200
```

Comment:
I feel like using `should` isn't correct.  I will check with @dkniffin or someone else and let you know.